docs/reference/env-variables.rst

```diff
@@ -0,0 +1,703 @@
+    *
+      - ROC_ACTIVE_WAIT_TIMEOUT
+      - uint
+      - 0
+      - Forces active wait of GPU interrup for the timeout, us unit
+    *
+      - ROC_ENABLE_LARGE_BAR
+      - bool
+      - TRUE
+      - Enable Large Bar if supported by the device
+    *
+      - ROC_CPU_WAIT_FOR_SIGNAL
+      - bool
+      - TRUE
+      - Enable CPU wait for dependent HSA signals.
+    *
+      - ROC_SYSTEM_SCOPE_SIGNAL
+      - bool
+      - TRUE
+      - Enable system scope for signals, uses interrupts.
+    *
+      - GPU_FORCE_QUEUE_PROFILING
+      - bool
+      - FALSE
+      - Force command queue profiling by default
+    *
+      - HIP_MEM_POOL_SUPPORT
+      - bool
+      - FALSE
+      - Enables memory pool support in HIP
+    *
+      - HIP_MEM_POOL_USE_VM
+      - bool
+      - IS_WINDOWS
+      - Enables memory pool support in HIP
+    *
+      - PAL_HIP_IPC_FLAG
+      - bool
+      - FALSE
+      - Enable interprocess flag for device allocation in PAL HIP
+    *
+      - PAL_FORCE_ASIC_REVISION
+      - uint
+      - 0
+      - Force a specific asic revision for all devices
+    *
+      - PAL_EMBED_KERNEL_MD
+      - bool
+      - FALSE
+      - Enables writing kernel metadata into command buffers.
+    *
+      - ROC_GLOBAL_CU_MASK
+      - cstring
+      - 
+      - Sets a global CU mask , entered as hex value for all queues, Each active bit represents using one CU , e.g., 0xf enables only 4 CUs
+    *
+      - AMD_LOG_LEVEL_FILE
+      - cstring
+      - 
+      - Set output file for AMD_LOG_LEVEL, Default is stderr
+    *
+      - PAL_PREPINNED_MEMORY_SIZE
+      - size_t
+      - 64
+      - Size in KBytes of prepinned memory
+    *
+      - AMD_CPU_AFFINITY
+      - bool
+      - FALSE
+      - Reset CPU affinity of any runtime threads
+    *
+      - ROC_USE_FGS_KERNARG
+      - bool
+      - TRUE
+      - Use fine grain kernel args segment for supported asics
+    *
+      - ROC_P2P_SDMA_SIZE
+      - uint
+      - 1024
+      - The minimum size in KB for P2P transfer with SDMA
+    *
+      - ROC_AQL_QUEUE_SIZE
+      - uint
+      - 16384
+      - AQL queue size in AQL packets
+    *
+      - ROC_SIGNAL_POOL_SIZE
+      - uint
+      - 32
+      - Initial size of HSA signal pool
+    *
+      - DEBUG_CLR_LIMIT_BLIT_WG
+      - uint
+      - 16
+      - Limit the number of workgroups in blit operations
+    *
+      - ROC_SKIP_KERNEL_ARG_COPY
+      - bool
+      - FALSE
+      - If true, then runtime can skip kernel arg copy
+    *
+      - GPU_STREAMOPS_CP_WAIT
+      - bool
+      - FALSE
+      - Force the stream wait memory operation to wait on CP.
+    *
+      - HIP_USE_RUNTIME_UNBUNDLER
+      - bool
+      - FALSE
+      - Force this to use Runtime code object unbundler.
+    *
+      - HIPRTC_USE_RUNTIME_UNBUNDLER
+      - bool
+      - FALSE
+      - Set this to true to force runtime unbundler in hiprtc.
+    *
+      - HIP_INITIAL_DM_SIZE
+      - size_t
+      - 8 * Mi
+      - Set initial heap size for device malloc.
```

Comment:
```suggestion
      - 8388608
      - Set initial heap size for device malloc. The default value corresponds to 8 MiB
```
I'm quite sure, that if the type is `size_t`, the default value is not `8 * Mi`. I just assumed, that the unit is bytes.